src/USBHID-Keyboard/KeyboardLayout_JP.h

```diff
@@ -0,0 +1,199 @@
+/*
+ * Japanese 106 keyboard layout.
+ */
+
+#include "KeyboardLayout.h"
+
+#pragma once
+
+//Additional keys
+#define KEY_J_YEN (136+0x89)
+
+#define AMOUNT_OF_SPECIAL_CHARS 2
```

Comment:
There is only 1 special char in the list

Reply:
I fixed the language code and special char count.